doc/content/manuals/theory/index.md

```diff
@@ -1,21 +1,49 @@
+semi-infinite domains, fault rupture model for seismic source
+simulation, and the domain reduction method for the input of complex,
+three-dimensional wave fields are incorporated. Since MASTODON is a <a href="http://mooseframework.org/"> MOOSE </a>-based
+application, it can efficiently solve problems using standard
+workstations or very large high-performance computers.
+
+This document describes the theoretical and numerical foundations of
+MASTODON.
 
 ## Governing equations
 The basic equation that MASTODON solves is the nonlinear wave equation:
 
 \begin{equation}
 \label{eqn:governing_equation}
+<<<<<<< de2079b461234601978c3cd2e70cb5c6fbcfd7f3
  \rho \mathbf{\ddot{u}} + \nabla \cdot \sigma = \mathbf{F_{ext}}
 \end{equation}
 
 where, $\rho$ is the density of the soil or structure that can vary with space, $\sigma$ is the stress at any point in space and time, $\mathbf{F_{ext}}$ is the external force acting on the system that can be in the form of localized seismic sources or global body forces such as gravity, and $\mathbf{\ddot{u}}$ is the acceleration at any point within the soil-structure domain. The left side of the equation contains the internal forces acting on the system with first term being the contribution from the inertia, and the second term being the contribution from the stiffness of the system. Additional terms would be added to this equation when damping is present in the system. The material stress response ($\sigma$) is described by the constitutive model, where the stress is determined as a function of the strain ($\epsilon$), i.e. $\sigma(\epsilon)$. Details about the material constitutive models available in MASTODON are presented in the section about [material models](#Material models).
+=======
+\rho \mathbf{\ddot{u}} + \nabla \cdot \boldsymbol{\sigma} = \mathbf{F_{ext}}
+\end{eqnarray}
+
+where $\rho$ is the density of the soil or structure that can vary with space, $\boldsymbol{\sigma}$ is the stress at any point in space and time, $\mathbf{F_{ext}}$ is the external force acting on the system that can be in the form of localized seismic sources or global body forces such as gravity, and $\mathbf{\ddot{u}}$ is the acceleration at any point within the soil-structure domain. The left side of the equation contains the internal forces acting on the system with first term being the contribution from the inertia, and the second term being the contribution from the stiffness of the system. Additional terms would be added to this equation when damping is present in the system. The material stress response ($\boldsymbol{\sigma}$) is described by the constitutive model, where the stress is determined as a function of the strain ($\boldsymbol{\epsilon}$), i.e. $\boldsymbol{\sigma}(\boldsymbol{\epsilon})$. Details about the material constitutive models available in MASTODON are presented in the section about [Material models](##Material models).
+>>>>>>> Changes to theory manual, additions to user and examples manuals #3
```

Comment:
The earlier paragraph from commit de2079b461234601978c3cd2e70cb5c6fbcfd7f3 should be kept and the above paragraph should be removed.